code/modules/reagents/chemistry/readme.md

```diff
@@ -122,6 +122,19 @@ Reagents are all the things you can mix and fille in bottles etc. This can be an
 			reacts with a mob. The method var can be either
 			TOUCH or INGEST. You'll want to put stuff like
 			acid-facemelting in here.
+		
+		on_mob_add(mob/living/L)
+			Called when the reagent datum is created in the mobs holder.
+			Adding more while they still have some dosn't triger the effect a second time.
```

Comment:
This line seems to be missing something, e.g. an E and a G.  Should probably be "doesn't trigger".